src/Controller/AjaxController.php

```diff
@@ -52,30 +52,31 @@ public function handle($key, $input = '') {
     }
   }
 
-    protected function stateProvince($input) {
-        if (!$input || ((int) $input != $input && $input != 'default')) {
-            $data = ['' => t('- first choose a country')];
-        }
-        else {
-            $data = \Drupal::service('webform_civicrm.utils')->wf_crm_get_states($input);
-        }
-
-        // @todo use Drupal's cacheable response?
-        return new JsonResponse($data);
+  protected function stateProvince($input, $isBilling = FALSE) {
+    if (!$input || ((int) $input != $input && $input != 'default')) {
+      $data = ['' => t('- first choose a country')];
+    }
+    else {
+      $data = \Drupal::service('webform_civicrm.utils')->wf_crm_get_states($input, $isBilling);
     }
 
-    protected function county($input) {
-        $data = [];
-        $utils = \Drupal::service('webform_civicrm.utils');
-        if (strpos($input, '-') !== FALSE) {
-            list($state, $country) = explode('-', $input);
-            $params = [
-              'field' => 'county_id',
-              'state_province_id' => $utils->wf_crm_state_abbr($state, 'id', $country)
-            ];
-            $data = $utils->wf_crm_apivalues('address', 'getoptions', $params);
-        }
-        // @todo use Drupal's cacheable response?
-        return new JsonResponse($data);
+    // @todo use Drupal's cacheable response?
+    return new JsonResponse($data);
+  }
+
+  protected function county($input, $isBilling = FALSE) {
+    $data = [];
```

Comment:
`county`? Does `county` make it into CiviCRM Billing Fields?

Reply:
I think this was added to make the function params compatible with [how it is called](https://github.com/colemanw/webform_civicrm/blob/8.x-5.x/src/Controller/AjaxController.php#L46). I agree it looks a bit odd. will fix in the next update. Thanks for pointing 👍